### PR TITLE
Nü Onboarding: Mark II

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
- Wires Simperium Mark 0.8.23
- Implements the new Onboarding UI (Initial ViewController)
- Nukes Legacy Onboarding + Unused assets
- Brings new Images + Colors!
- Wires the new Onboarding UI using the **new** Simperium mechanisms

Ref. #359

### Testing:
- [x] Verify the Onboarding UI cannot be launched, directly, in landscape mode
- [x] Verify the UI matches [the mockups!](https://user-images.githubusercontent.com/1195260/61151709-f30c7e80-a4bc-11e9-8ac8-468bee7bcefd.png)

Both **Signup** / **LogIn** are disabled in this PR. Coming up next, brand new Authentication UI!

### CODE REVIEW

File: Simplenote/Classes/UIColorName.swift [hunk 1]
```diff
@@ -100,6 +101,8 @@ extension UIColorName {
             return .searchHighlightFontColor
         case .simplenoteAlmostBlack:
             return .simplenoteAlmostBlack
+        case .simplenoteGunmetal:
```
Comment (bummytime) [inline Simplenote/Classes/UIColorName.swift]: Out of scope for this PR, but I am wondering if we should eventually adopt the color studio names for the colors we use...

File: Simplenote/Classes/SPOnboardingViewController.swift [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    func setupLabels() {
+        let simplenoteAlmostBlack = UIColor.color(name: .simplenoteAlmostBlack)
+
+        simplenoteLabel.text = OnboardingStrings.brandText
+        simplenoteLabel.textColor = simplenoteAlmostBlack
+
+        headerLabel.text = OnboardingStrings.headerText
+        headerLabel.textColor = simplenoteAlmostBlack
+
+        if #available(iOS 11, *) {
```
Comment (bummytime) [inline Simplenote/Classes/SPOnboardingViewController.swift]: Does this iOS 11+ check need to be here?

Comment (jleandroperez) [reply Simplenote/Classes/SPOnboardingViewController.swift]: I'm afraid it does! Our deployment target is set to iOS 10. Nuking this triggers a warning.

Eventually we should bump up the target!

File: Simplenote/Classes/UIColorName.swift (hunk 1 above)
Comment (jleandroperez) [reply Simplenote/Classes/UIColorName.swift]: Studio Names == Zeplin right?

I've **tried** to adopt names similar to iOS Dynamic Fonts. `Color for Header` etc etc etc. BUT those new names come from the Zeplin mockups. I'd go with whatever is easier to maintain, definitely!!

**Comment by jleandroperez** [comment]
Thank you for reviewing this one sir!!! WOOOOO!!!

🔥

**Pull request merged**